lib/keywords.js

```diff
@@ -81,3 +81,8 @@ export const loadDialect = (dialectId, dialect, allowUnknownKeywords = false) =>
       }
     });
 };
+
+export const getDialectIds = () => {
+  return Object.keys(_dialects);
+};
```

Comment:
I prefer the inline style for small one-liners like this.

```suggestion
export const getDialectIds = () => Object.keys(_dialects);
```